MCprep_addon/conf.py

```diff
@@ -207,6 +208,54 @@ def deprecation_warning(self):
 			self.log("Deprecation Warning: This will be removed in MCprep 3.5.1!")
 			traceback.print_stack()
 
+class ErrorType(enum.Enum):
+	"""
+	All error types
+
+	Attributes
+	------------
+	FILE_NOT_FOUND:
+		File was not found
+	"""
+	FILE_NOT_FOUND = enum.auto()
```

Comment:
I feel like error type is unnecessary, we already have a whole slew of exception classes that python provides. Why not make that the datatype field, a _type_ assignment, e.g. something like this in the dataclass: `error_type = Exception` (not sure how typing would be for that actually, since it shouldn't be an _instance_ of an exception)

Reply:
I think it should be fine to use exception classes directly, since they're just objects. As long as they're not being raised, it shouldn't cause any problems

Reply:
Yup exactly, let's do that - will avoid a lot of re-definitions of things, I think